app/javascript/shared/components/emoji/EmojiInput.vue

```diff
@@ -174,7 +174,7 @@ export default {
       return categoryItem ? categoryItem.emojis[0].emoji : '';
     },
     focusSearchInput() {
-      this.$refs.searchbar.focus();
+      this.$refs.searchbar?.focus();
```

Comment:
Can we use `autofocus` property on the input instead?

Reply:
@scmmishra I tried adding the `autofocus` property for input, but it doesn't work.

Reply:
Were you able to reproduce it?. Try adding a `$nextTick` see if that fixes the issue.

Reply:
@scmmishra No, I couldn't able to reproduce it.

Reply:
@scmmishra Updated the PR

Reply:
Sounds good, if this occurs again, we will now. I will update the sentry rule